src/mcts/search.cc

```diff
@@ -83,6 +83,7 @@ namespace {
 void ApplyDirichletNoise(Node* node, float eps, double alpha) {
   float total = 0;
   std::vector<float> noise;
+  noise.reserve(node->GetNumEdges());
```

Comment:
My tests indicate this doesn't make a difference.

Reply:
I was able to measure an improvement on my low end laptop but not on my high end desktop.

Reply:
That's a very slow laptop. Dirichlet noise is only applied once per search, it wouldn't be noticeable even on computer which is 25 years old.

Reply:
I think I'm going to give up benchmarking anything of this laptop. I think has some dramatic temperature controls that make it unreliable for benchmarking.